jwst/assign_wcs/nirspec.py

```diff
@@ -142,6 +147,7 @@ def imaging(input_model, reference_files):
                             (sca, det2gwa),
                             (gwa, gwa2msa),
                             (msa_frame, None)]
+        # TODO: Add DVA correction
```

Comment:
There's no need to add VA correction here since this pipeline does not go to sky but stops at the MSA.